.github/CODEOWNERS

```diff
@@ -103,7 +103,7 @@ README*                                         @ceph/doc-writers
 /qa/workunits/rbd                               @ceph/rbd
 /src/ceph-rbdnamer                              @ceph/rbd
 /src/cls/journal                                @ceph/rbd
-/src/cls/lock                                   @ceph/rbd
+/src/cls/lock                                   @ceph/rbd @ceph/rgw
```

Comment:
fyi @idryomov; i *think* this is the right way to share ownership of cls_lock

Reply:
Yeah, it probably is.  Thanks, I opened https://github.com/ceph/ceph/pull/50076 in a similar vein!